server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -99,15 +99,18 @@ export function create(
 		authorization: string,
 		params: IWholeSummaryPayload,
 		initial?: boolean,
+		initialStorageName?: string,
```

Comment:
Why not use `storageName`? we don't have a different storage name for an initial summary.